target/idl/token_bridge_relayer.json

```
@@ -1,5 +1,5 @@
 {
-  "address": "ttbrcA1ckR3D3Ff4VR1MJNCvA7t4d4XV9TcvrVp4AoM",
+  "address": "4whSAdxwWa4b4TQZFamJJJ1pthJPMFmeMHqjzWUutSjP",
```

It's better to keep the address as is. I think it's set to the local testnet address.
```suggestion
  "address": "ttbrcA1ckR3D3Ff4VR1MJNCvA7t4d4XV9TcvrVp4AoM",
```